app/gateway-oss/2.4.x/hybrid-mode.md

```diff
@@ -324,6 +324,18 @@ cluster_cert_key = cluster.key
 lua_ssl_trusted_certificate = cluster.crt
 ```
 
+### DP Node Stat Sequence
+
+When set as a DP node, Kong will follow the following steps regarding its configuration: 
+
+step | description
+---|---
+config cache | If a file `config.json.gz` exists in the `kong_prefix` path (`/usr/local/kong` by default), the DP Node loads it as configuration.
+declarative_config | If there wasn't a config cache and the `declarative_config` parameter is set, it's loaded as usual.
+empty config | If neither the config cache or a declarative configuration is available, the Node starts with an empty configuration.  In this state it will return 404 to all requests.
+contact CP Node | In all cases, the DP Node contacts the CP Node to retrieve the latest configuration.  If successful, it will be stored in the local config cache (`config.json.gz`)
```

Comment:
```suggestion
contact CP Node | In all cases, the DP Node contacts the CP Node to retrieve the latest configuration. If successful, it will be stored in the local config cache (`config.json.gz`)
```